PowerEditor/src/MISC/PluginsManager/PluginsManager.cpp

```diff
@@ -78,11 +78,11 @@ int PluginsManager::loadPlugin(const TCHAR *pluginFilePath, vector<generic_strin
 	    pi->_hLib = ::LoadLibrary(pluginFilePath);
         if (!pi->_hLib)
         {
-            const std::wstring& lastErrorMsg = GetLastErrorAsString();
+            generic_string lastErrorMsg = GetLastErrorAsString();
             if (lastErrorMsg.empty())
                 throw generic_string(TEXT("Load Library is failed.\nMake \"Runtime Library\" setting of this project as \"Multi-threaded(/MT)\" may cure this problem."));

```

Comment:
> any good tool to manage

Git or TortoiseGit and probably others I don't know of.

> communication channel

See Community and Gitter badges on [this page](https://github.com/notepad-plus-plus/notepad-plus-plus) right under the this [header](https://github.com/notepad-plus-plus/notepad-plus-plus#what-is-notepad-).